docs: add logs_body_key_attributes option for opentelemetry output (#2055)
fluent/fluent-bit-docs: pipeline/outputs/opentelemetry.md

| `logs_span_id_metadata_key`            | Specify a `SpanId` key` to look up in the metadata.           | `$SpanId`         |
| `logs_trace_id_metadata_key`           | Specify a `TraceId` key to look up in the metadata.          | `$TraceId`        |
| `logs_attributes_metadata_key`         | Specify an `Attributes` key to look up in the metadata.      | `$Attributes`     |

> | `logs_body_key_attributes`             | When enabled (`true`), Fluent Bit will place all parsed JSON fields (except the one mapped by `logs_body_key`) inside the `attributes` section of the OpenTelemetry `LogRecord`. This is useful if you want structured key/value pairs as attributes, while still mapping a specific field (defined by `logs_body_key`) to the `body`. | `false` |

Reviewer: ```suggestion
| `logs_body_key_attributes`             | When enabled (`true`), Fluent Bit places all parsed JSON fields (except the one mapped by `logs_body_key`) inside the `attributes` section of the OpenTelemetry `LogRecord`. Use to create structured key/value pairs as attributes, while still mapping a specific field (defined by `logs_body_key`) to the `body`. | `false` |
```

Just trying to make it a little more active and avoid "useful". Looks good to me though!

@fluent/fluent-bit-maintainers should review for technical accuracy.
